django_extensions/db/models.py

```diff
@@ -25,6 +25,7 @@ def save(self, **kwargs):
     class Meta:
         get_latest_by = 'modified'
         ordering = ('-modified', '-created',)
+        indexes = [models.Index(fields=["created"]), models.Index(fields=["modified"])]
```

Comment:
Currently these indexes are like adding `db_index=True` to each of the fields (`created` and `modified`). There is no need for `Meta.indexes` in this case. 

If `Meta.indexes` should be defined I thing that is better to add index which is for both fields in descending order (to match the `Meta.ordering`). This way listing of the objects should be faster. Also individual indexes can also be in descending order (at least for `modified`) for `latest()` to use it.

Reply:
I agree with your points. So do you think it would be better if the modified the current version like this:
```
indexes = [ 
    models.Index(fields=["-created", "-modified"]),
    models.Index(fields=["-created"]), models.Index(fields=["-modified"])
]
```